src/OpenAI.csproj

```diff
@@ -46,6 +46,6 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="8.0.0" PrivateAssets="All" />
     <PackageReference Include="System.ClientModel" Version="1.1.0-beta.4" />
-    <PackageReference Include="System.Text.Json" Version="8.0.2" />
```

Comment:
Why was this one deleted? STJ was only added to the runtime in .NET 6 so the netstandard2.0 target would fall back to System.ClientModel's dependency on STJ 4.7.2. I don't know if that is desirable.

Reply:
Reverted

Reply:
But it was deleted in another PR